TLM/TLM/UI/Textures/RoadUI.cs

```diff
@@ -69,5 +69,20 @@ public static class RoadUI {
             VehicleInfoSignTextures[ExtVehicleType.Taxi] = LoadDllResource("RoadUI.taxi_infosign.png", signSize);
             VehicleInfoSignTextures[ExtVehicleType.Tram] = LoadDllResource("RoadUI.tram_infosign.png", signSize);
        }
+
+        public static void ReloadTexturesWithTranslation() {
+            IntVector2 size = new IntVector2(200);
```

Comment:
wtf is this? :D
... we should use something like ValueTuple<T1, T2> instead.
but ok .. not part of this PR.

Reply:
This is like Vector2 but for ints with exactly same meaning, it is x and y, or width and height together. Are tuples even supported in the C# version we're using? Why would I want a tuple or a new generic class name for something that only is used for width;height?

Reply:
`ValueTuple<>` is .net 4.7+ so... nope :)

Reply:
yeah it is .. but its easy to implement our own.

Reply:
@kvakvs tuples are just generic structs. sure they are supported by c#.

Reply:
Then you can assume i implemented our own. It carries meaning of width;height pair

Reply:
https://github.com/CitiesSkylinesMods/TMPE/pull/1122